pl_bolts/models/detection/faster_rcnn/faster_rcnn_module.py

```diff
@@ -117,7 +103,7 @@ def validation_step(self, batch, batch_idx):
         images, targets = batch
         # fasterrcnn takes only images for eval() mode
         outs = self.model(images)
-        iou = torch.stack([_evaluate_iou(t, o) for t, o in zip(targets, outs)]).mean()
+        iou = torch.stack([_evaluate_iou(o, t) for t, o in zip(targets, outs)]).mean()
```

Comment:
To keep consistency with `evaluate_iou`

Reply:
why you have args `o, t` but zip `targets, outs` instead of `outs, targets`
the also rather call out as predits

Reply:
@oke-aditya mind following the @Borda's comment?

Reply:
Tbh either doesn't matter. IoU is commutative.
Also let's merge this and refactor the metrics to use torchvision in a new PR.